install_config/install/docker_registry.adoc

```diff
@@ -169,9 +170,68 @@ $ oadm registry --service-account=registry \
 ----
 endif::[]
 
-[[viewing-logs]]
 
+[[registry-ip-address]]
+=== Registry IP Address
+
+Because the integrated registry is referenced by its service IP address,
+if you decide to delete and recreate the docker-registry service,
+you should either take care to re-use its address,
+or reboot some or all of the cluster after creating the new docker-registry service.
+
+[[re-using-the-address]]
+Re-using the Address::
+
+To re-use the IP address, you must save the IP address of the old docker-registry
+service prior to deleting it, and arrange to replace the newly assigned IP address
+with the saved one in the new docker-registry service.
+
+// NB: Snarfed from <https://github.com/openshift/openshift-docs/issues/1494>.
+. Make a note of the `ClusterIP` for the service.
+
+. Delete the service.
```

Comment:
@miminar 
Thanks for the tip.  I think the second command is easier to understand on quick reading, and have added it.